development/slider-carousel-app/pages/slider-carousel/index.mdx

```diff
@@ -0,0 +1,59 @@
+---
+HEAD OBJECT
+title: "ReuseJS slider-carousel"
+author: "Md Amir Sohail"
+---
+import SliderWithButton from "../../components/slider-carousel/Example/SliderWithButton/SliderWithButton.tsx"
+import SimpleSlider from "../../components/slider-carousel/Example/SimpleSlider/SimpleSlider.tsx"
+import SliderWithLinearSlide from "../../components/slider-carousel/Example/SliderWithLinearSlide/SliderWithLinearSlide.tsx"
+
+## Table of contents
+
+## **Introduction**
+>Take your website to the next level with ReuseSliderCarousel. Effortlessly create stunning image carousels and captivating slideshows. Customize slide intervals, enable looping, and add intuitive button controls. Engage your visitors with seamless transitions and eye-catching animations. Showcase your content in style and leave a lasting impression.
+
+## ReuseSliderCarousel Component Docs
+The `ReuseSliderCarousel` component is a reusable carousel component that displays a collection of slides. It provides functionality for `automatic slide transitions`, `navigation buttons`, and `customizable styles`.
+
+- React component that renders a slider carousel using the `ReuseSliderCarousel` component from the `@locoworks/reusejs-react-slider-carousel` library. The slider displays a list of slides, and it automatically transitions between slides at a specified interval.
+
+- The slider component from `ReuseSliderCarousel` has the feature to go to the next or previous slide when clicked, based on the click position. It detects the click position relative to the `slider container` and determines whether the user clicked on the left or right side of the container.
+
+## Props
+- `slideInterval` (number): The interval (in milliseconds) between slide transitions.
+- `slides` (React.ReactNode[]): An array of ReactNode elements representing the slides to be displayed.
+- `loop` (boolean): A boolean value indicating whether the carousel should loop back to the first slide after reaching the last slide.
+- `wrapperClasses` (string): Additional CSS classes to be applied to the wrapper container element.
+- `sliderContainerClasses` (string): Additional CSS classes to be applied to the slider container element.
+- `enableButtons` (boolean): A boolean value indicating whether navigation buttons should be displayed.
+- `prefixButton` (React.ReactNode): Custom ReactNode element to be used as the prefix button. Defaults to a left arrow icon.
+- `suffixButton` (React.ReactNode): Custom ReactNode element to be used as the suffix button. Defaults to a right arrow icon.
+- `buttonClasses` (string): Additional CSS classes to be applied to the navigation buttons.
+- `animationStyle` (string): Determines the animation style of the slider. Set this prop to 'continue' to enable continuous looping with slide chaining. When set to 'continue', the slider will seamlessly transition from the last slide to the first slide, creating a continuous loop effect.
+
+## Examples
+### **Slider without button**
```

Comment:
Remove ** ### already makes it a heading

Reply:
ok,
removed ** and used only ### for that purpose Thank You.